.circleci/config.yml

```diff
@@ -40,25 +40,15 @@ jobs:
       - run:
           name: Run backend tests
           command: |
-            docker-compose -f ./test/docker-compose.yml up -d
+            docker-compose -f ./test/docker-compose.yml up test
```

Comment:
They‘re started because lf the depends_on, but I think that‘s the issue: We need to wait until the DBs are fully running, not just starting. Will investigate